main.go

```diff
@@ -121,3 +134,21 @@ func validatedPrefixes(prefixes []string) ([]string, error) {
 
 	return validatedPrefixes, nil
 }
+
+func getPrefixesWriteFunc(config *Config, currentNamespace string) (prefixcollector.WritePrefixesFunc, error) {
+	switch config.PrefixesOutputType {
+	case fileOutputType:
+		return prefixwriter.NewFileWriter(config.OutputFilePath), nil
+	case configMapOutputType:
+		return prefixwriter.NewConfigMapWriter(config.NSMConfigMapName, currentNamespace), nil
+	default:
+		return nil, errors.New("Wrong prefixes output")
```

Comment:
I would also suggest adding method Validate for the Config

Reply:
Added validate method